### DISCUSSION TITLE
asking to add support for this usb wifi Edimax EW-7711MAC AC450 Nano USB Wi-Fi

### DISCUSSION THREAD

**Discussion by b1t0y** [chris1111/Wireless-USB-Big-Sur-Adapter#276] [General]
Can you add support for this usb wifi Edimax EW-7711MAC AC450 Nano USB Wi-Fi? What do you need from me in able to have support for this usb wifi? Thanks

**Comment by chris1111** [comment]
This is a Mediatek Drivers, it is support here
[D-LinkUtility-Package](https://github.com/chris1111/D-LinkUtility-Package)

**Reply by b1t0y** [reply]
Thank you so much for this
